Dockerfile

```diff
@@ -1,16 +1,7 @@
-FROM node:carbon
-
-# Create app directory
-RUN mkdir -p /app
+FROM node:current
```

Comment:
Should we lock down the version number? I think `:current` defaults to the most recent node image.